docs/src/appendix/forge-library/deserialize_txt.md

```diff
@@ -0,0 +1,53 @@
+# `deserialize_txt`
+
+
+> `trait TxtParser<T, impl TSerde: Serde<T>> {
+>     fn deserialize_txt(file: @File) -> Option<T>;
+> }` 
+
+Parses plain text file content and tries to deserialize it to type `T` that implements `Serde` trait.
+
+- `file` - a snapshot of an instance of the struct `File` that consists of the following fields:
+  - `path` - Cairo shortstring representing a path to a file relative to a package root.
+
+```rust
+use option::OptionTrait;
+use serde::Serde;
+use snforge_std::{ FileTrait, TxtParser };
+
+#[derive(Serde, Drop)]
+struct MyStruct {
+    a: u32,
+    b: felt252
+}
+
+#[test]
+fn test_deserialize_txt() {
+    let file = FileTrait::new('data/file.txt');
+    let my_struct = TxtParser::<MyStruct>::deserialize_txt(@file).unwrap();
+    // ...
+}
+```
+
+File content must consists of elements that:
+- have to be separated with whitespaces
+- have to be either:
+  - integers in range of `[0, P)` where P is [`Cairo Prime`](https://book.cairo-lang.org/ch02-02-data-types.html?highlight=prime#felt-type)
+  - strings of length `<=31` enclosed in single or double quotation marks
```

Comment:
why do we allow for both single and double quotation marks? Shouldn't we wait with double until strings are supported?

Reply:
I did that only to be in line with future json support, as json standard does not allow single quoted strings. Not sure if it is a good idea though cc @MaksymilianDemitraszek

Reply:
I think we could add double quotes in the json PR

Reply:
I think we should use single quutation for short strings, double quoutations should be used for long strings once those are released.